manifests/Gaguka.yaml

```
@@ -0,0 +1,10 @@
+name: Gaguka
+authors: MIMSBZ3
+homepage: https://github.com/MIMSBZ3/Gaguka
+license: GPL-3.0-or-later
+version: v0.1
```

Missing a download URL

```suggestion
version: v0.1
url: https://github.com/MIMSBZ3/Gaguka/archive/refs/tags/v0.1.zip
```